scripts/verify_merges.sh

```diff
@@ -0,0 +1,67 @@
+#! /bin/bash
+
+#    Copyright (C) 2014-2018 Bill Winslow
+#    Copyright (C) 2017-2018 Christian Beer
+#
+#    This script is a part of the mfaliquot package.
+#
+#    This program is libre software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+#    See the LICENSE file for more details.
+
+aliqueit="./aliqueit"
+mergefile="./allseq.merges.txt"
+errfile="./allseq.broken.txt"
+emailscript="/usr/bin/env python3 ./send_email.py"
+
+if [[ ! -s $mergefile ]]; then exit 1; fi
+
+if [[ ! -f $aliqueit || ! -x $aliqueit ]]; then
+	$emailscript "Could not find aliqueit executable at $aliqueit"
+fi
+
+for line in $(cat $mergefile); do
+	error=0
+	allelf=""
+	# read $line into array $seqs using _ as delimiter
+	readarray -td_ seqs <<<"${line}_"; unset 'seqs[-1]';
```

Comment:
What's the purpose of the unset there? Every sequence listed in the line should be verified

Reply:
Well as my other comment below demonstrates, all sequences are in fact tested, so although I remain ignorant of the exact working of this line, there's no problem with it